src/main/java/tech/thatgravyboat/repolib/api/EnchantsAPI.java

```diff
@@ -0,0 +1,34 @@
+package tech.thatgravyboat.repolib.api;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+public final class EnchantsAPI {
+
+    private final Map<String, JsonObject> enchantments = new HashMap<>();
```

Comment:
these should have an internal representation like runes and pets that gets made from the json.

Reply:
but other things dont have that 🥺